01-JS-DOM-APIs/EX01/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
```

Comment:
**Minor**: Fix the identation here. I don't remember if I told you about minor comments. They are related to have a clean code, the severity associated is low, you can skip this type of comments.